.github/ISSUE_TEMPLATE.md

```diff
@@ -2,16 +2,22 @@ Welcome to Ghost-CLI's GitHub repo! 👋🎉
 
 Do you need help or have a question? Please come chat in Slack: https://slack.ghost.org/ 👫.
 
-### This issue is a
+Docs: https://docs.ghost.org 📖.
+
+Please be aware that the team behind the Ghost CLI only recommended stack: https://github.com/TryGhost/Ghost-cli#recommended-stack.
```

Comment:
This seems off to me... is there a word missing? 🤔

Reply:
Good 🎣 , thanks!